docs/community/governance.md

```diff
@@ -1,76 +1,84 @@
-# Loki's Governance Model
+# Governance
 
-This document describes the rules and governance of the Loki project. It is meant to be followed by all the developers of the project and the community.
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Loki community. Common terminology used in this governance document are listed below: 
+
+* **Team members**: Any members of the private [team mailing list][team].
+
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][maintainers]).
+
+* **Projects**: A single repository in the Grafana GitHub organization and listed below is referred to as a project:
+  * tanka
```

Comment:
not sure why tanka is called out here, should this be Loki?

Reply:
Fixed, thanks